src/components/GroupCard/GroupCard.js

```diff
@@ -52,7 +53,9 @@ export default function GroupCard ({
         />
         {group.description
           ? <div styleName='group-description'>
-            <HyloHTML element='span' html={TextHelpers.markdown(group.description)} />
+            <ClickCatcher groupSlug='all'>
```

Comment:
why is groupSlug always all, shouldn't it be the group slug?

Reply:
Sorry it's a long description, but all true: 

My goal in pushing this change is simply to get external links handled in the group description the same way they are handled everywhere else. 

It will only come-up if someone puts in a mention or a topic in the description, which there basically isn't an obvious way for a user to do right now. 

The Group Description entry probably now be given a `HyloEditor` instead of a plain text markdown interpreted input, but I didn't want to get into that now either and it wasn't something I was trying to fix with this. 

\* Putting the base slug as that of the group being described means that in many cases the user won't yet be a member of that group at all, so if there were a mention or a topic linked here in the description tied to that group context, it could be inaccessible to the user viewing the description when it maybe wouldn't be otherwise? 

Assuming `currentGroup` would probably be arbitrary to this content, so not that. 

The default for the `ClickCatcher` in its few pieces is 'all' anyway for topics and mentions and I think that is all the slug is used for in the catcher. Outsides links don't care, and relative Hylo links will just be the actual path with no regard for this prop. So actually the `groupSlug` prop should just be left off in all these cases and added if/when we find cases where we want a different result down the road. For now I included as it felt safer and more explicit to do so.

Reply:
Furthermore, and I need to create a ticket for this, the markdown transformations happening client side here and with Messages#text should all be moved to the backend and sanitized on the way out. I imagined that work happening at the same time we get around to dropping `HyloEditor` in as the input. It will need to be thought through then what we want to do with the existing markdown content as the new or edited entries will be stored as HTML once we are using the editor. Probably we'll just materialize the previous content to HTML in a migration.

Reply:
FYI - created these two follow-up tickets which maybe should happen in one fell swoop but would be fine to one-by-one as well: 

https://github.com/Hylozoic/hylo-evo/issues/1349
https://github.com/Hylozoic/hylo-evo/issues/1350

Reply:
gotcha, makes sense!